siddhar_iyal/constellation_names.eng.fab

```diff
@@ -0,0 +1,43 @@
+Gdn	"Garudan"	_("கருடன்")
```

Comment:
As mentioned before, this formatting makes the file untranslatable for all but your culture. We could host it for optional download, but it will certainly not go into the main package.  

As described in the User Guide, the required format is

`Abbr "OriginalNameInOriginalGlyphs" _("Translatable name in European characters")`

The preferred format that helps all translators and lets other people learn from your contribution is:

`Abbr "OriginalNameInOriginalGlyphs" _("Name or transliteration in European characters (English translation)")
`